src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -359,10 +359,12 @@ def help_text
       # @return [Boolean] true when skipping has been confirmed
       def show_skipping_warning
         # Popup question: confirm skipping the registration
-        warning = _("If you do not register your system we will not be able\n" \
-            "to grant you access to the update repositories.\n\n" \
-            "You can register after the installation or visit our\n" \
-            "Customer Center for online registration.")
+        warning = _("Without registration, update-channels will not be\n" \
+            "configured. This disables updates and security fixes.\n\n" \
+            "A full system can be installed using the\n" \
+            "SLE-15-Packages media from download.suse.com.\n" \
+            "Without this media only a minimum system is available\n" \
```

Comment:
"medium"

Reply:
Hm, I have asked a native speaker to check these strings and she has approved it.

Reply:
It's a bit complicated. It's actually a set of 3 media, but in reality just the DVD1 is enough (DVD2 = sources, DVD3 = debug packages IIRC).

Reply:
OK, if we are talking about several pieces of storage media, then "these media".